src/DynamoUtilities/VersionUtilities.cs

```diff
@@ -62,5 +62,67 @@ public static Version Parse(string version)
             // Now it should be safe to parse
             return Version.Parse(version);
         }
+
+        /// <summary>
+        /// Parse the first n fields of a version string.  Delegates to
+        /// Version.Parse.
+        /// </summary>
+        public static Version WildCardParse(string version)
+        {
+            // If the version string is null or empty, return null
+            if (string.IsNullOrEmpty(version))
+            {
+                return null;
+            }
+
+            // Check if the version string ends with a wild card.
+            if (!version.EndsWith(".*"))
+            {
+                return null;
+            }
+
+            var splitVersion = version.Split('.');
+
+            //CHeck that there are not more then 3 version items specified
+            if (splitVersion.Length > 3)
+            {
+                return null;
+            }
+
+            // Check that the first number is a valid number
+            if (!int.TryParse(splitVersion[0], out _))
+            {
+                return null;
+            }
+
+            if (splitVersion.Length == 2)
+            {
+                var newVersion = splitVersion[0] + ".1000.0";
```

Comment:
Maybe 1000 is not enough.... Could be more

Reply:
Ahh we should probably bump this to a bit higher just to be really out of range.  Maybe 99999?

Reply:
@zeusongit maybe we need to adjust the rules on versions

Reply:
Just to confirm here in writing and for posterity @saintentropy - the `major/minor/patch` maximum value for .Net is 2,147,483,647. The `revision` is 65,535.